src/stores/PostStore.ts

```diff
@@ -32,14 +30,13 @@ interface PostStoreType {
   idToPostTx: Promise<string[]>
 }
 
-const farcasterContract = getPostStorage()
+export const farcasterContract = getPostStorage()
```

Comment:
Why do we need to do it like so? Can't we use `getPostStorage()` in other place to get `farcasterContract`?

Reply:
Changed it